### PR TITLE
chore: cleanup unused index for notification

### PR DESCRIPTION
**Pull request by guqing**
#### What type of PR is this?
/kind cleanup
/area core
/milestone 2.19.x

#### What this PR does / why we need it:
清理没有用到的索引

#### Does this PR introduce a user-facing change?
```release-note
None
```

### CODE REVIEW

**Review by JohnNiang — APPROVED** [review]
/approve

**Review by ruibaby — APPROVED** [review]
/lgtm

File: application/src/main/java/run/halo/app/notification/UserNotificationQuery.java [hunk 1]
```diff
@@ -29,72 +16,26 @@
...
-        return sort.and(Sort.by(
-            Sort.Order.desc("metadata.creationTimestamp"),
-            Sort.Order.desc("metadata.name"))
-        );
-    }
-
     /**
      * Build a list options from the query object.
      */
     public ListOptions toListOptions() {
```
Comment (JohnNiang) [inline application/src/main/java/run/halo/app/notification/UserNotificationQuery.java:31]: 当前方法似乎重载了 SortableRequest#toListOptions，建议加上 `@Override` 注解。

**Review by JohnNiang — CHANGES_REQUESTED** [review]

File: application/src/main/java/run/halo/app/notification/UserNotificationQuery.java (hunk 1 above)
Comment (guqing) [reply application/src/main/java/run/halo/app/notification/UserNotificationQuery.java:32]: Done

**Review by JohnNiang — APPROVED** [review]
/lgtm

**Pull request merged**